draft-pda-protocol.md

```diff
@@ -561,7 +566,8 @@ aggregator.]
 
 The leader responds to well-formed requests to `[leader]/upload` with status 200
 and an empty body. Malformed requests are handled as described in
-{{pa-error-common-aborts}}.
+{{pa-error-common-aborts}}. Clients MUST NOT upload the same measurement value in
+more than one report if the leader responds with status 200 and an empty body.
```

Comment:
It seems like this would require clients to maintain a list of uploaded reports to avoid double-uploads, which contradicts the later statement:
>Clients are not expected to store any state across any upload flows

Besides which, I think leaders will need to implement detection of duplicate report uploads no matter what, in which case they can return an error to clients in the case of a double upload, which downgrades this to a SHOULD NOT.

Reply:
Yeah, this seems reasonable.